Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -468,8 +471,8 @@ FullDiagnostics::PrepareFieldDataForOutput ()
     warpx.FillBoundaryAux(warpx.getngUpdateAux());
 
     // Update the RealBox used for the geometry filter in particle diags
-    for (int i = 0; i < m_output_species.size(); ++i) {
-        m_output_species[i].m_diag_domain = m_geom_output[0][0].ProbDomain();
+    for (int i = 0; i < m_output_species[0].size(); ++i) {
```

Comment:
Let's add a comment here as well why we only access `[0]`

Reply:
generalized this. Thank you!